app/src/lib/web-flow-committer.ts

```diff
@@ -31,9 +31,19 @@ export function isWebFlowCommitter(
     return true
   }
 
-  if (name === 'GitHub Enterprise') {
-    const host = new URL(endpoint).host.toLowerCase()
-    return email.endsWith(`@${host}`)
+  if (endpoint !== getDotComAPIEndpoint() && name === 'GitHub Enterprise') {
```

Comment:
I think this is safe, we could I'm not sure if email starts with `ghe-noreply` would also work or be better/worse?

Reply:
It looks like that part is configurable as well unfortunately.

Reply:
🤯